dropbox/oauth.py

```diff
@@ -146,22 +175,29 @@ def _get_authorize_url(self, redirect_uri, state, token_access_type, scope=None,
             assert token_access_type in TOKEN_ACCESS_TYPES
             if token_access_type != 'legacy':
                 params['token_access_type'] = token_access_type
+        if code_challenge and code_challenge_method:
+            assert code_challenge_method in PKCE_CODE_CHALLENGE_METHOD_TYPES
+            params['code_challenge'] = code_challenge
+            params['code_challenge_method'] = code_challenge_method
 
         if scope is not None:
             params['scope'] = " ".join(scope)
-        if include_granted_scopes is not None:
-            assert include_granted_scopes in INCLUDE_GRANTED_SCOPES_TYPES
-            params['include_granted_scopes'] = str(include_granted_scopes)
+            if include_granted_scopes is not None:
```

Comment:
If we decide to do this, worth adding validation in constructor that 
`if include_granted_scopes: 
    validate_not_empty(scope)
`